app/controllers/articles_controller.rb

```diff
@@ -3,11 +3,13 @@ class ArticlesController < ApplicationController
 
   # GET /articles or /articles.json
   def index
-    @articles = Article.all
+    @pagy, @articles = pagy(Article.all, items: 10)
```

Comment:
would be good to move "magic number" into a constant, and make it possible to change via `params`, e.g. use constant when params does not have `per_page` option